spyder/widgets/switcher.py

```diff
@@ -269,6 +269,10 @@ def __init__(self, parent=None, icon=None, title=None, description=None,
 
         if icon:
             self.setIcon(icon)
+            # TODO: Change fixed icon size value
+            self._icon_width = 20
```

Comment:
So this should be updated based on the editor zoom maybe?

Reply:
I was thinking to get the size from the icon but I'm unsure how to get it